File: UICollectionView_p1_ObjC/UICollectionView_p1_ObjC/Views/GalleryItemCollectionViewCell.m
//
//  GalleryItemCollectionViewCell.m
//  UICollectionView_p1_ObjC
//
//  Created by olxios on 19/11/14.
//  Copyright (c) 2014 olxios. All rights reserved.
//

#import "GalleryItemCollectionViewCell.h"
#import "GalleryItem.h"

@implementation GalleryItemCollectionViewCell

- (void)setGalleryItem:(GalleryItem *)item
{
    _itemImageView.image = [UIImage imageNamed:item.itemImage];
}

@end

File: UICollectionView_p1_Swift/UICollectionView_p1_Swift/Views/GalleryItemCollectionViewCell.swift
//
//  GalleryItemCollectionViewCell.swift
//  UICollectionView_p1_Swift
//
//  Created by olxios on 20/11/14.
//  Copyright (c) 2014 olxios. All rights reserved.
//

import UIKit

class GalleryItemCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet var itemImageView: UIImageView!
    
    func setGalleryItem(item:GalleryItem) {
        itemImageView.image = UIImage(named: item.itemImage)
    }
    
}